qcodes_contrib_drivers/drivers/Lakeshore/Model_625.py

```diff
@@ -0,0 +1,523 @@
+import logging
+import time
+from typing import Union, Tuple
+
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import  Numbers, Enum
+
+log = logging.getLogger(__name__)
```

Comment:
I think this can safely be removed as `self.log` is used

Reply:
Ok, I removed `log = logging.getLogger(__name__)`